terraform/environment/region/modules/uploads_s3_bucket/lambda.tf

```diff
@@ -66,7 +66,7 @@ resource "aws_scheduler_schedule" "invoke_lambda_every_15_minutes" {
     mode = "OFF"
   }
 
-  schedule_expression = "rate(20 minutes)"
+  schedule_expression = "rate(60 minutes)"
```

Comment:
maybe the name(s) of this resource should also not mention a time?

Reply:
good point, it's 3 or 4 changes old. fixed